src/util/include/util/utf8_view.h

```diff
@@ -60,12 +60,21 @@ struct utf8_view: std::ranges::view_interface<utf8_view<InputIterator, InputSent
         return std::u8string_view(reinterpret_cast<const char8_t*>(b.it), reinterpret_cast<const char8_t*>(e.it));
     }
 
+    constexpr auto c_str() const -> const char8_t*
+        requires(std::same_as<Sentinel, SentinelImpl> && InputSentinel{} == u8"")
+    {
+        return static_cast<const char8_t*>(beginI);
+    }
+
```

Comment:
So, the point of utf8_view is that casting to `const char8_t*` is undefined behaviour, so this function is a no-go...